teuthology/task/install/rpm.py

```diff
@@ -46,8 +46,8 @@ def _remove(ctx, config, remote, rpm):
         if dist_release in ['opensuse', 'sle']:
             _zypper_removerepo(remote, repos)
         else:
-            raise Exception('Custom repos were specified for %s ' % remote_os +
-                            'but these are currently not supported')
```

Comment:
This change is not related to the issue, so could you please revert it. Besides using `.format` everywhere has no reasonable background and does not help to be python3 compliant.